store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10560,4 +10560,14 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Feature to enable delivery status notification</desc>
 </attr>
+
+<attr id="4136" name="zimbraFeatureExternalEmailWarningEnabled" type="boolean" cardinality="single" optionalIn="cos,account" flags="accountInherited,accountInfo" since="10.1.7">
+  <defaultCOSValue>FALSE</defaultCOSValue>
+  <desc>Feature to enable/disable External Email Warning (EEW)</desc>
+</attr>
+
+<attr id="4137" name="zimbraExternalEmailWarningMessage" type="string" cardinality="single" optionalIn="globalConfig" since="10.1.7">
```

Comment:
can different domain/cos set this value differently?

Reply:
We want to set this warning message as same globally, not differently for cos or domain.

Reply:
@silentsakky  Setting it at global,domain level